### ISSUE TITLE
[TSDoc> Search]: Screen Reader is not conveying 'ctrl+k' shortcut keys information present on the Search button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/rushstack-websites#243]
**Test Environment:**
Version: Version 24H2 (OS Build 26231.5000)
Browser Version: 127.0.2638.2 (Official build) dev (64-bit)
URL: https://tsdoc.org/
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL in edge dev browser and login with v-.
2. Turn on the narrator through Ctrl+ windows+ Enter key.
3. Navigate to the 'Search' button and observe the issue.

Actual Result:
Screen reader is not conveying the shortcut keys information which is mentioned on the search button. While navigating on it, screen reader is conveying as **'Search Button.'**

**Expected Result:**
Screen Reader should announce the mentioned shortcut key on the search button. When the focus lands on it then screen reader should announce as **'Search button user ctrl+k key to activate it.'**

**Note:**
Same issue is repro through NVDA as well.

**User Impact:**
It will impact the screen reader user as they will not able to know the shortcut key to activate the control.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

![Screen Reader is not conveying 'ctrl+k' shortcut keys information present on the Search button](https://github.com/microsoft/rushstack-websites/assets/95913748/7c29082b-fe30-46b0-9729-e5eb152c524a)

https://github.com/microsoft/rushstack-websites/assets/95913748/2d27caee-2916-4c5b-8ac0-33badc920ee5
